README.md

```diff
@@ -17,6 +17,17 @@ Returns an array with personalization data.
 ### returnVaryHeader($key)
 Returns vary header array, based on header data.
 
+## Global Methods
```

Comment:
For the function documentation, I think we omit this section and just document the static functions as the recommended way to interact with these functions. In so doing, though, I think adding usage examples to the method docblocks within `HeaderData.php` would be a good addition -- so there's _some_ documentation for how to use the class methods directly (if for whatever reason you didn't want to use the static methods), but you have to go looking for it.

Something like:

```php
  /**
   * Gets a header by key.
   *
   * @param string $key
   *   Key for the header.
   *
   * Example:
   *     $headers = new Pantheon\EI\HeaderData();
   *     $headers->getHeader( 'Audience' );
   *
   * @return string
   *   Returns header value.
   */
```

Reply:
Added some inline example calls to the relevant global functions.

Reply:
This section in the readme can be removed and the public functions be moved up to replace the function names and docs above.